Update migration page and small fixes (#1776)
smartcontractkit/documentation: src/content/chainlink-automation/guides/migrate-to-v2.mdx

<ClickToZoom src="/images/automation/v2-migration/migration-txn.png" />

### Migrating multiple upkeeps

If you have multiple upkeeps to migrate, you can do so in the Chainlink Automation App.

> 1. Click the link displays on your upkeeps dashboard. This link only displays if you have one or more upkeeps to migrate:

Reviewer: ```suggestion
1. Click the link displays on your upkeeps dashboard. This link displays only if you have one or more upkeeps to migrate:
```

   <ClickToZoom src="/images/automation/v2-migration/1-bulk-migration-ui.png" />

> 1. Follow the prompts to approve and confirm the transactions to migrate your upkeep(s).

Reviewer: ```suggestion
1. Follow the prompts to approve and confirm the transactions to migrate your upkeeps.
```

   <ClickToZoom src="/images/automation/v2-migration/approve-migration.png" />

> ### Updating permissions

Reviewer: Is this required for bulk migration? If yes, add a step to point users to this section instructing them to update their permissions. Also, does this apply to the "Migrating using the Chainlink Automation App" section?

Author: It potentially applies to any type of migration, so I restructured it to reflect that. Thanks!

1. Execute the `migrateUpkeeps` function and confirm the transaction. When this transaction is confirmed, your upkeeps will be migrated to the latest registry, and each upkeep will have a [unique forwarder](#unique-forwarder) address.

> 1. Update your contract to use the forwarder address by following the instructions in the [Forwarder](/chainlink-automation/guides/forwarder) page.

Reviewer: ```suggestion
1. Update your contract to use the forwarder address by following the instructions on the [Forwarder](/chainlink-automation/guides/forwarder) page.
```

> When the migration is complete:

Reviewer: ```suggestion
After the migration is complete:
```

- Your balance is transferred to the new registry automatically.

> - Read the [migration checklist](#migration-checklist) to understand further updates you may need to make in your contracts.

Reviewer: ```suggestion
- Read the [migration checklist](#migration-checklist) to understand further updates you might need to make in your contracts.
```
